src/routes/Home/index.jsx

```
@@ -11,31 +14,59 @@ const storedTags = [];
 
 export default function Home() {
   const navigate = useNavigate();
+  const { user, logout } = useAuth(); // ContextAPI에서 유저 정보 가져오기
```

ESLint

[error] 'logout' is assigned a value but never used.

(no-unused-vars)